src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      data-qa="big" 
+      action="http://localhost:8080/" 
```

Comment:
When I send my request, search-bar redirecting me to page not found, try `action=:"/"`